### PR TITLE
JBPM-9286 UI error when parent process instance details have been rem…

### PR DESCRIPTION
**Pull request by bxf12315**
…oved, add unit test

**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[link](https://www.example.com)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* paste the link(s) from GitHub here
* link 2
* link 3 etc.

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: kie-server-parent/kie-server-remote/kie-server-client/src/test/java/org/kie/server/client/QueryServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    @Test
+    public void testFindProcessesByIdNotFound() {
+        stubFor(get(urlEqualTo("/"))
+                        .withHeader("Accept", equalTo("application/xml"))
+                        .willReturn(aResponse()
+                                            .withStatus(404)
+                                            .withHeader("Content-Type", "application/xml")
+                                            .withBody("<response type=\"NOTFOUND\" msg= </response>")));
+
+        List<ProcessDefinition> list = queryServicesClient.findProcessesById("Not eixst");
```
Comment (nmirasch) [inline kie-server-parent/kie-server-remote/kie-server-client/src/test/java/org/kie/server/client/QueryServiceImplTest.java:37]: @bxf12315 typo in "Not eixst"

**Review by elguardian — APPROVED** [review]
Aside from the typo LGTM

**Review by nmirasch — APPROVED** [review]

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by lazarotti** [comment]
for all: this is an additional PR sent, to add Tests as asked by Cristiano. As the others PRs are merged this should not block the Jiras linked to this PR due to the cutoff, even though we should work to have this extra tests included in our code base later anyway

**Comment by mareknovotny** [comment]
jenkins retest this please

do we still want to merge this new test?

**Review by mareknovotny — APPROVED** [review]

**Comment by mareknovotny** [comment]
@bxf12315 please check if this is added also in 7.67.x/7.67.x-blue branches if not back port it there

**Comment by bxf12315** [comment]
Done

**Pull request merged**